contributing/ci-ome-files.txt

```diff
@@ -0,0 +1,170 @@
+OME Files jobs
+--------------
+
+All jobs are listed under the :jenkinsview:`Files` view tab of Jenkins.
+
+.. list-table::
+        :header-rows: 1
+
+        -       * Job task
+                * Development series (0.x.y)
+
+        -       * Builds the latest OME Files C++ components (Unix superbuild)
+                * :term:`OME-FILES-CPP-DEV-latest-superbuild`
+
+        -       * Builds the latest OME Files C++ components (Windows superbuild)
+                * :term:`OME-FILES-CPP-DEV-latest-win-superbuild`
+
+        -       * Builds the latest OME Files C++ documentation
+                * :term:`OME-FILES-CPP-DEV-latest-docs`
+
+        -       * Triggers the building of the merge OME Files C++ components (all merge build jobs)
+                * :term:`OME-FILES-CPP-DEV-merge-trigger`
+
+        -       * Merges the ome-cmake-superbuild PRs
+                * :term:`OME-FILES-CPP-DEV-merge-push-superbuild`
+
+        -       * Builds the merge OME Files C++ components (Unix)
+                * :term:`OME-FILES-CPP-DEV-merge`
+
+        -       * Builds the merge OME Files C++ source releases (Unix)
+                * :term:`OME-FILES-CPP-DEV-merge-sourcebuild`
+
+        -       * Builds the merge OME Files C++ components (Unix superbuild)
+                * :term:`OME-FILES-CPP-DEV-merge-superbuild`
+
+        -       * Builds the merge OME Files C++ components (Windows superbuild)
+                * :term:`OME-FILES-CPP-DEV-merge-win-superbuild`
+
+        -       * Builds the merge OME Files C++ documentation
+                * :term:`OME-FILES-CPP-DEV-merge-docs`
+
+Stable series
+^^^^^^^^^^^^^
+
+There is not currently a stable (1.0) release of OME Files.
+
+Development series
+^^^^^^^^^^^^^^^^^^
+
+The branch for the development series of OME Files is develop.
+
+.. glossary::
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-latest-superbuild`
+
+                This job builds the develop branches of OME Files components (Unix)
+
+                #. |buildFilesSB|
+
+            See :jenkinsjob:`the build graph <OME-FILES-CPP-DEV-latest-superbuild/lastSuccessfulBuild/BuildGraph>`
```

Comment:
The 'See' isn't rendered on the page here, or for the following 2 instances or for OME-Files-CPP-DEV-merge-docs - I think it's because you need to indent it further, the instance on line 88 below renders properly and that's indented inline with the other points.